ipa-core/src/report/hybrid.rs

```diff
@@ -262,12 +498,14 @@ where
     const CIPHERTEXT_MK_OFFSET: usize = Self::ENCAP_KEY_MK_OFFSET + EncapsulationSize::USIZE;
     const ENCAP_KEY_BTT_OFFSET: usize = (Self::CIPHERTEXT_MK_OFFSET
         + TagSize::USIZE
-        + <Replicated<BA64> as Serializable>::Size::USIZE);
+        //+ <Replicated<BA64> as Serializable>::Size::USIZE);
+        + Replicated::<BA64>::size());
     const CIPHERTEXT_BTT_OFFSET: usize = Self::ENCAP_KEY_BTT_OFFSET + EncapsulationSize::USIZE;
 
     const KEY_IDENTIFIER_OFFSET: usize = (Self::CIPHERTEXT_BTT_OFFSET
         + TagSize::USIZE
-        + <Replicated<BK> as Serializable>::Size::USIZE);
+        //+ <Replicated<BK> as Serializable>::Size::USIZE);
```

Comment:
Lets clean up commented code before merging it

Reply:
Ah sorry, missed that one